synapseutils/copy.py

```diff
@@ -344,9 +369,15 @@ def copyWiki(syn, entity, destinationId, entitySubPageId=None, destinationSubPag
                 wNew = Wiki(owner=newOwn, title=wiki.get('title',''), markdown=wiki.markdown, attachments=attachments, parentWikiId=wikiIdMap[wiki.parentWikiId])
```

Comment:
elif wiki['attachmentFileHandleIds'] != []:
                 uri = "/entity/%s/wiki/%s/attachmenthandles" % (wiki.ownerId, wiki.id)
                 results = syn.restGET(uri)

is already available as     def getWikiAttachments(self, wiki):